draft-sheffer-oauth-rfc8725bis.md

```diff
@@ -681,8 +687,14 @@ As discussed in  {{use-typ}}, for new JWT
   RECOMMENDED.
 
 
+## Limit Hash Iteration Count {#limit-iterations}
 
-
+Implementations are RECOMMENDED to set a reasonable upper limit on
+the number of hash iterations that can be performed
+when validating encrypted content using PBES2 encryption algorithms,
+so as to prevent attackers from imposing
+an unreasonable computational burden on recipients.
+Rejecting inputs with a `p2c` (PBES2 Count) value over 10,000 is RECOMMENDED.
```

Comment:
1M?

https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html

Reply:
I thought 1M was what was recemmended as well

Reply:
You're looking at outdated text.  Look at the files changed.  It's now 1.2M - double the NIST value.

Reply:
I am looking at the files changed -- at least I think I am! If 1.2M then I'm fine.